docs/configuration/settings.rst

```diff
@@ -403,6 +403,23 @@ Form settings
 Contact settings
 ****************
 
+Multiple company management
+===========================
+
+ .. image:: images/multiple-companies-enabled.png
+      :width: 600
+      :alt: Screenshot showing the Multiple Companies feature enabled
+
+This setting, when enabled, allow a Contact to link with more than one Company. It's beneficial for businesses that interact with clients across multiple Companies.
+ 
+When you turn this off, a Contact can only link to one Company. This is ideal for businesses with simpler structures where each Contact only needs a single Company association.
+
```

Comment:
Do we need the extra line break here?

Reply:
@RCheesley  No, we don't need it. cc: @fakela